src/scripting/game_lua_kernel.cpp

```diff
@@ -103,6 +103,7 @@
 #include "units/map.hpp"  // for unit_map, etc
 #include "units/ptr.hpp"                 // for unit_const_ptr, unit_ptr
 #include "units/types.hpp"    // for unit_type_data, unit_types, etc
+#include "formula/string_utils.hpp"    // for unit_type_data, unit_types, etc
```

Comment:
i think theses comments were added by an automatic toll and commited by iceiceice. Afaik the main point was to make it more obviou when includes can be removed to make cleaning up includes easier.